azure-functions/acbs-function/jest.common.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  preset: 'ts-jest',
+  transform: {
+    '^.+\\.js$': 'babel-jest',
+    '^.+\\.ts$': 'ts-jest',
+  },
+  moduleNameMapper: {
+    '^.+\\.(css|less|scss)$': 'babel-jest',
+    uuid: require.resolve('uuid'),
```

Comment:
Do we need `uuid`? Can't see any instances of this in the `azure-functions` directory, maybe could be the same as TFM UI's `jest.common.config`

Reply:
good spot